tests/common.go

```diff
@@ -4967,6 +4967,41 @@ func StartTorpedoTest(testName, testDescription string, tags map[string]string,
 	tags["pureVolume"] = fmt.Sprintf("%t", Inst().PureVolumes)
 	tags["pureSANType"] = Inst().PureSANType
 	dash.TestCaseBegin(testName, testDescription, strconv.Itoa(testRepoID), tags)
+	SetClusterOptions()
```

Comment:
Adding here will make this call every time a new test is started.
can we move it before suite?
Also enabling by default is there any possibility of missing issues when auto fstrim is disabled

Reply:
Modified to call before suite,
Torpedo default APPPS: elasticsearch,postgres,sysbench,nginx-sharedv4,vdbench-sharedv4,nginx-sv4-svc,vdbench-sv4-svc 
We are going to enable FSStrim on "elastic search, nginx-sharedv4, sysbench, nginx-sv4-svc, postgres, vdbench-sv4-svc 
We still have APP "vdbench-sharedv4" which is not enabled for autofsTrim. if you want I can disable autofstrim on additional apps, idea is to improve missing test coverage in Torpedo as part of EPIC 
https://portworx.atlassian.net/browse/PTX-13937

Reply:
Test logs  https://jenkins.pwx.dev.purestorage.com/job/Users/job/Santosh/job/BasicTorpedo/19/
https://jenkins.pwx.dev.purestorage.com/job/IPv6/job/tp-ipv6-basic-csi-CentOS-79/141/console